packages/core/src/renderer.ts

```diff
@@ -779,6 +881,73 @@ export class Renderer {
         };
         break;
       }
+      case "web3.eth.personal.sign": {
+        const promise = new Promise<void>((resolve) => {
+          setTimeout(() => {
+            this.onEthPersonalSignAction(
+              {
+                data: {
+                  // domain: this.replaceInlineContext(action.data.domain),
+                  // address: this.replaceInlineContext(action.data.address),
+                  statement: this.replaceInlineContext(action.data.statement),
+                  // uri: this.replaceInlineContext(action.data.uri),
+                  version: this.replaceInlineContext(action.data.version),
+                  chainId: this.replaceInlineContext(action.data.chainId),
```

Comment:
Should `domain`, `address`, `uri` not be exposed?

Reply:
this is done by the implementing client in the action handler

Reply:
ah okay makes sense